go.mod

```diff
@@ -26,29 +26,29 @@ require (
 	github.com/integr8ly/cloud-resource-operator v1.0.1
 	github.com/integr8ly/keycloak-client v0.1.11
 	github.com/noobaa/noobaa-operator/v5 v5.11.0
-	github.com/onsi/gomega v1.20.1
+	github.com/onsi/gomega v1.23.0
 	github.com/openshift/api v3.9.1-0.20191031084152-11eee842dafd+incompatible
-	github.com/operator-framework/api v0.17.1
+	github.com/operator-framework/api v0.17.3
 	github.com/operator-framework/operator-lifecycle-manager v0.21.1
```

Comment:
@adam-cattermole , I'd like to ask - if need move to latest version of operator-lifecycle-manager - https://github.com/operator-framework/operator-lifecycle-manager/releases/tag/v0.24.0  , as you have commit to address [MGDAPI-4898]. 
Moving from operator-framework/operator-lifecycle-manager/pkg/api/apis/operators to operator-framework/api/pkg/operators looks very good. 
Just wanted to ask about operator-lifecycle-manager v0.21.1, although it used just in 2 ".go" files + hardcoded in some several MCG CSVs, and maybe more. But maybe it's part of this task. Thank you